docs/framework/develop-client-apps.md

```diff
@@ -73,9 +73,9 @@ Here's a few trade-offs to consider. These applications run only in Windows 10 a
 
 :heavy_check_mark: Simple, clean, and trustworthy deployment to Windows 10 users by using the Desktop Bridge.
```

Comment:
@merriemcgaw - should we mention ClickOnce deployment as well since we mentioned win7 in the first point? ClickOnce deploys to all supported versions of windows.

Reply:
It does seem off not to have ClickOnce mentioned since deployment via Desktop Bridge is listed as a reason to use Windows apps.  You could simply add "and using ClickOnce to deploy to legacy Windows versions" or something to that effect.  It's reasonable to deemphasize ClickOnce on Win10 since we're trying to push customers towards Desktop Bridge/Store based deployments.  FWIW though ClickOnce is fully supported on Win10 and from what I've seen so far easier than using Desktop Bridge to deploy.

Reply:
@normesta can you look at this feedback?